### PR TITLE
Kl page

### PR DESCRIPTION
**Pull request by kimleangkheng**

### CODE REVIEW

File: developer/language/reference/ethnologuecode.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Description
+
+The optional `&EthnologueCode` store specifies the language the keyboard
+is made for by use of an ISO 639-3 three-letter language code. This
+information is intended for use by third-party applications, and is also
+used by the compiler to generate metadata for KeymanWeb.
+
+For a list of codes, or more information about Ethnologue, see
+[www.ethnologue.com](http://www.ethnologue.com/){.ulink
```
Comment (Nnyny) [inline developer/language/reference/ethnologuecode.md:30]:
```suggestion
[www.ethnologue.com](http://www.ethnologue.com/)
```

File: developer/language/reference/ethnologuecode.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+target="_blank"}.
```
Comment (Nnyny) [inline developer/language/reference/ethnologuecode.md:31]:
```suggestion
.
```

File: developer/language/reference/includecodes.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### Parameters
+
+`filename`
+:   The file to import, relative to keyboard source path.
+
+## Description
+
+The `&includecodes` store allows you to reference an external file for
+[named character constants](../guide/constants){.link}. Keyman Developer
```
Comment (Nnyny) [inline developer/language/reference/includecodes.md:24]:
```suggestion
[named character constants](../guide/constants). Keyman Developer
```

**Review by Nnyny — CHANGES_REQUESTED** [review]

File: developer/language/reference/ethnologuecode.md (hunk 2 above)
Comment (jahorton) [reply developer/language/reference/ethnologuecode.md:31]: Good catch!  The `target="_blank"` would admittedly be nice to keep... if it worked.  

I just tested adding that in to a different link on my machine... alas, I don't think we can use that feature.  So, good catch; we do need to remove both parts within the `{` and `}`.

**Review by jahorton — APPROVED** [review]

**Pull request merged**